connector-doc-generator/resources/prompt-template-phase1.md

```
@@ -0,0 +1,288 @@
+<file name="setup-guide.md">
+...markdown content... (only if service-side setup steps exist)
+</file>
+
+<file name="trigger-reference.md">
+...markdown content... (only if has_triggers is true)
+</file>
+
+<category_entry>
+{"description": "one-line connector description for the catalog index table", "operations": "Create, Read, Update, Delete", "auth": "OAuth 2.0"}
+</category_entry>
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)